packages/mobile/src/components/News.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {View, Text, Image, StyleSheet, ActivityIndicator} from 'react-native';
+
+const News = (props) => {
+	return(
+		<View style={ styles.newsItem }>
+			{ props.data.urlToImage ? <Image
+				style={ styles.newsImage }
+				source={{ uri: props.data.urlToImage }}
+				PlaceholderContent={<ActivityIndicator />}
```

Comment:
Czym jest PlaceholderContent, bo umnie pokazuje ze takie coś nie istnieje, sprawdź to prosze.